src/mono/wasm/build/WasmApp.targets

```diff
@@ -134,7 +134,26 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_WasmGenerateAppBundle" Condition="'$(WasmGenerateAppBundle)' == 'true'">
+  <Target Name="_WasmGenerateRuntimeConfig">
+    <PropertyGroup>
+      <RuntimeConfigFilePath>$(TargetDir)publish\$(AssemblyName).runtimeconfig.json</RuntimeConfigFilePath>
+      <ParsedRuntimeConfigFilePath>$(TargetDir)publish\runtimeconfig.bin</ParsedRuntimeConfigFilePath>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(RunAOTCompilation)' == 'true'">
+      <RuntimeConfigReservedProperties Include="RUNTIME_IDENTIFIER"/>
+      <RuntimeConfigReservedProperties Include="APP_CONTEXT_BASE_DIRECTORY"/>
+    </ItemGroup>
+
+    <!-- Parse *.runtimeconfig.json file -->
+    <RuntimeConfigParserTask
```

Comment:
Corresponding to this, the `RuntimeConfigParser` task should error out if the config file does not exist.

Reply:
Added error checking.